dashboards-observability/public/components/common/search/autocomplete.test.tsx

```diff
@@ -105,7 +107,7 @@ describe('autocomplete logic', function () {
         itemName: 'source',
       },
     ] as AutocompleteItem[];
-    const suggestion = await getFullSuggestions(input, dslService);
+    const suggestion = await getFullSuggestions('', input, dslService);
```

Comment:
I don't have any context on autocomplete, why have the function parameters changed to take in an empty string base query?

Reply:
These tests are checking if `getFullSuggestions` is working correctly which is a function that isn't called from Application Analytics so the base query will always be an empty string.

Reply:
The reason I have to add the `base` parameter that doesn't get used anyway is that the function for getting suggestions is passed as a prop to the autocomplete component and the other function `getSuggestionsAfterSource` does need that parameter.

Reply:
But as I'm writing this I'm realizing that it may be better to have the prop have two different types and just remove the unnecessary parameter from the function.

Reply:
Just tried and it is actually very ugly, may just keep dummy parameter

Reply:
I see, if you are going to keep the dummy parameter can you add a comment detailing why?